python/ambassador/envoy/v3/v3cluster.py

```diff
@@ -203,19 +215,27 @@ def get_endpoints(self, cluster: IRCluster):
 
         if len(targetlist) > 0:
             for target in targetlist:
-                address = {
-                    "address": target["ip"],
-                    "port_value": target["port"],
-                    "protocol": "TCP",  # Yes, really. Envoy uses the TLS context to determine whether to originate TLS.
+                endpoint = {
+                    "address": {
+                        "socket_address": {
+                            "address": target["ip"],
+                            "port_value": target["port"],
+                            "protocol": "TCP",  # Yes, really. Envoy uses the TLS context to determine whether to originate TLS.
```

Comment:
Is this comment still relevant, if not I would say remove it. Because, personally it makes no sense to me so does additional context need to be added or since we are modifying the code we should just remove it.